_std/pathfinding.dm

```diff
@@ -452,6 +456,9 @@
 		if(!wall.broken && (F.floorrunning || (F.can_floorrun && F.resources >= 10))) //greater than 10 to give some wiggle room, actual cost is 1 per wall tile
 			return TRUE // floor running drones can *always* pass through flockwalls
 
+	if(T.passability_cache == null || options[POP_IGNORE_CACHE])
+	else
+		return T.passability_cache // not anymore
```

Comment:
```suggestion
	if(T.passability_cache != null && !options[POP_IGNORE_CACHE])
		return T.passability_cache
```

Reply:
fwiw ![wanderer](https://cdn.discordapp.com/attachments/890226559691157524/1040906108212088894/image.png)

Reply:
![image](https://user-images.githubusercontent.com/65367576/204355548-b878350b-c0f8-45ec-8776-40c51eab3041.png)
undid the sanity. long live chaos

Reply:
Seriously, though, this is actually insignificant on a grand scale, it saves like 1 nanosecnd and can actually cost 30-90 nanoseconds. Which is literally nothing, even though this is the most critical pathway in the entire codebase by call count.
Feel free to change it to be more readable